wdl/Utils.wdl

```diff
@@ -357,3 +489,113 @@ task LocalizeCloudFileWithCredentials {
   }
 }
 
+task WriteLines {
```

Comment:
This is unfortunate (that you had to do it, I mean)

Reply:
Deleted also, looks like I managed to remove this from the workflows at some point